datadog_checks_dev/datadog_checks/dev/docker.py

```diff
@@ -117,6 +167,24 @@ def docker_run(
     if conditions is not None:
         docker_conditions.extend(conditions)
 
+    if mount_logs:
+        if isinstance(mount_logs, dict):
+            wrapper = shared_logs(mount_logs['logs'])
```

Comment:
This keyword is already part of the docker run method signature - should there be a check if its None and either log a warning or wrap the wrapper to avoid overwriting it?

Reply:
https://github.com/DataDog/integrations-core/pull/5361